org.mwc.cmap.core/src/org/mwc/cmap/core/ui_support/swt/SWTCanvasAdapter.java

```diff
@@ -1218,6 +1218,8 @@ public void setColor(final java.awt.Color theCol) {
 
 		if (theCol != _currentColor) {
 			_currentColor = theCol;
+			// Let's save the alpha in case the color has.

```

Comment:
Do you need to finish off the above sentence @saulhidalgoaular ?

Reply:
Do you mean to improve the grammar? I will change it in a sec.

Reply:
Solved in https://github.com/debrief/debrief/pull/5097/commits/445c6946693ffaa72be4a2121d44000db67ac7e9